design-documents/graph-ql/coverage/returns.graphqls

```diff
@@ -0,0 +1,197 @@
+type Query{
+    # See https://github.com/magento/architecture/blob/master/design-documents/graph-ql/framework/attributes-metadata.md
+    pageSpecificCustomAttributes(
+        page_type: CustomAttributesPageEnum!
+    ): CustomAttributeMetadata
+}
+
+type Mutation {
+    createReturn(input: CreateReturnInput!): CreateReturnOutput @doc(description: "Create a new return.")
```

Comment:
@paliarush since negotiable quotes also creates a request for the merchant to respond to, we can find similar naming.

I'm using [`requestNegotiableQuote`](https://github.com/magento/architecture/pull/399/files#diff-4d6fb9dda54abab9b58373010823458bR26) in that PR. We could do `requestReturn`.

Open to other suggestions, but unified naming would be good.